tools/scripts/bootstrap.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, args=None):
         self._project_root_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
         self._project_build_dir = os.path.join(self._project_root_dir, "build")
         self._sysroot_dir = os.path.join(self._project_build_dir, "sysroot")
-        self._cerbero_root_dir = os.path.join(self._external_cerbero_build_path or self._project_build_dir, "cerbero")
+        self._cerbero_root_dir = self._external_cerbero_build_path or os.path.join(self._project_build_dir, "cerbero")
```

Comment:
You may complete this change in method `copy_all_packages_from_external_cerbero_build` around line 170 by replacing:
```
if not os.path.exists(src):
    raise Exception(f"Unable to find package {src}")
```

by something like this:
```
if not os.path.exists(src):
    src = os.path.join(os.path.dirname(self._external_cerbero_build_path), filename)
    if not os.path.exists(src):
        raise Exception(f"Unable to find package {src}")
```

This way the script will work seamlessly with external cerbero builds where packages have been built in the same folder as the Cerbero one, and also with cerbero builds done thanks to the bootstrap.py script (in this case packages are built one folder up of the Cerbero one).

Reply:
copy step is not needed if cerbero is internal build for wpe-android. Added check for that to skip copy in that case